testsuite/features/support/constants.rb

```diff
@@ -129,7 +129,7 @@
 # For containers we do not have SCC, so we set the Fake Base Channel
 # for sle_minion
 sle_base_channel =
-  if ENV['PROVIDER'].include? 'podman'
+  if ENV['PROVIDER'].include?('podman') || ENV['PROVIDER'].include?('docker')
```

Comment:
There is a real need in the test framework to distinguish "podman" and "docker"?
Or should we name it "container" as a more generic name?

Reply:
Not anymore. We only support podman, so I have removed the "include docker" thing.